webtool/frontend/src/components/styles.css

```diff
@@ -0,0 +1,35 @@
+.form-box {
+  box-shadow: 1px 1px 5px #c3b6b6;
+}
+
+.radix-grid {
+  display: grid;
+  grid-template-columns: repeat(12, 0.1fr);
+}
+
+.form-group {
+  align-items: center;
+  margin-bottom: 16px;
+}
+
+.form-label {
+  font-size: 14px;
```

Comment:
(doesn't need to be changed now)

I'm of the opinion font size should mostly be 1 rem for accessibility and consistency across sites.

I this case it seems like we should configure this in Radix. I don't quite understand why the DataList elements are 14px now.